docs/docs.md

```diff
@@ -137,8 +139,9 @@ if the client is authenticated along with the reply that is sent when authentica
 
 | line(s)                                     | purpose                                                                                          |
 | ------------------------------------------- | ------------------------------------------------------------------------------------------------ |
-| [Lines 7 to 12](../config/webapp.rego#L7)   | Defines the error message sent if request authentication fails. |
-| [Lines 15 to 27](../config/webapp.rego#L15) | Sends a request to `/authenticated` on the example app while including all cookies. The example app validates the cookie session and returns `200 OK` when the request is authenticated, otherwise it returns `403 Forbidden`. The rego code checks the status code and returns `true` when the request is authenticated. |
+| [Line 6](../config/webapp.rego#L6)   | Reads the environment variable `EXAMPLE_APP_JWT_SECRET` into a variable with the default value `"Default JWT secret"`. The same default is used on the example app for ease of configuration but in a production quality application this should always be set from the environment variable. |
```

Comment:
```suggestion
| [Line 6](../config/webapp.rego#L6)   | Reads the environment variable, `EXAMPLE_APP_JWT_SECRET`, into a variable with the default value `"Default JWT secret"`. The same default is used on the example app for ease of configuration. In a production quality application, however, this should always be set from the environment variable. |
```
Two copy-edits:
* delimit the variable name with commas (ie, treat it as a parenthetical).
* restructured the 2nd sentence:
	* removed the conjunction, _but_;
	* turned the two independent clauses into sentences
	* added _however_ to function as a conjunctive adverb (ie, to explicitly express the implicit function provided by _but_).